e2e/scenario/scenario_ubuntu2204-standalone-containerd.go

```diff
@@ -0,0 +1,72 @@
+package scenario
+
+import (
+	"encoding/json"
+	"os"
+	"strings"
+
+	"github.com/Azure/agentbaker/pkg/agent/datamodel"
+)
+
+func (t *Template) ubuntu2204ContainerdURL() *Scenario {
+	return &Scenario{
+		Name:        "ubuntu2204ContainerdURL",
+		Description: "Tests that a node using the Ubuntu 2204 VHD with a containerd URL can be properly bootstrapped",
+		Config: Config{
+			ClusterSelector: NetworkPluginKubenetSelector,
+			ClusterMutator:  NetworkPluginKubenetMutator,
+			VHDSelector:     t.Ubuntu2204Gen2Containerd,
+			BootstrapConfigMutator: func(nbc *datamodel.NodeBootstrappingConfiguration) {
+				nbc.ContainerService.Properties.AgentPoolProfiles[0].Distro = "aks-ubuntu-containerd-22.04-gen2"
+				nbc.AgentPoolProfile.Distro = "aks-ubuntu-containerd-22.04-gen2"
+				nbc.ContainerdPackageURL = "https://packages.microsoft.com/ubuntu/22.04/prod/pool/main/m/moby-containerd/moby-containerd_1.6.9+azure-ubuntu22.04u1_amd64.deb"
+			},
+			LiveVMValidators: []*LiveVMValidator{
+				containerdVersionValidator("1.6.9"),
+			},
+		},
+	}
+}
+
+func (t *Template) ubuntu2204ContainerdVersion() *Scenario {
+	return &Scenario{
+		Name:        "ubuntu2204ContainerdVersion",
+		Description: "Tests that a node using the Ubuntu 2204 VHD with a containerd version can be properly bootstrapped",
+		Config: Config{
+			ClusterSelector: NetworkPluginKubenetSelector,
+			ClusterMutator:  NetworkPluginKubenetMutator,
+			VHDSelector:     t.Ubuntu2204Gen2Containerd,
+			BootstrapConfigMutator: func(nbc *datamodel.NodeBootstrappingConfiguration) {
+				nbc.ContainerService.Properties.AgentPoolProfiles[0].Distro = "aks-ubuntu-containerd-22.04-gen2"
+				nbc.AgentPoolProfile.Distro = "aks-ubuntu-containerd-22.04-gen2"
+				nbc.ContainerdVersion = "1.6.9"
+			},
+			LiveVMValidators: []*LiveVMValidator{
+				containerdVersionValidator(getCurrentManifestContainerdVersion()),
+			},
+		},
+	}
+}
+
+type Manifest struct {
+	Containerd struct {
+		Edge string `json:"edge"`
+	} `json:"containerd"`
+}
+
+func getCurrentManifestContainerdVersion() string {
```

Comment:
might be worth moving this to a common location so others can modify as necessary since it could be useful in other scenarios

Reply:
we could also just grab the manifest from the running node itself since it'll be there - I feel like that would be better since then we wouldn't have to rely on the manifest being in a particular location within the repo for this to work properly

this is probably okay for now but ^ that might be a good follow-up item (would basically just incorporate grabbing the manifest and checking the version into the live VM validator)